### ISSUE TITLE
404 on the bookmarklet page

### ISSUE THREAD

**Issue by peschee** [Heydon/REVENGE.CSS#29]
https://heydonworks.com/revenge_css_bookmarklet/ renders a 404:

![image](https://user-images.githubusercontent.com/63866/86444667-62ecc800-bd11-11ea-9808-9d81a8fd9268.png)

**Comment by JayHoltslander** [comment]
Was about to report the same.

**Comment by rubenvarela** [comment]
Same.
For reference, this is what was there, http://web.archive.org/web/20190611001226/http://heydonworks.com/revenge_css_bookmarklet

**Comment by trishalish** [comment]
I. Need a smart computer savvy and willing to break me off the truth about
my 14 year relationship. I'm the type that needs hard proof... Help

**Comment by Castria** [comment]
Same for me. Is this project abandoned?

**Comment by coliff** [comment]
> Is this project abandoned?

Yes I think so (unfortunately). Heydon has recommended on Twitter that users should check out a11y css by @ffoodd https://ffoodd.github.io/a11y.css/ instead.

**Comment by rubenvarela** [comment]
@coliff Thanks for that!

**Comment by niutech** [comment]
Any updates @Heydon?

**Comment by gregdaynes** [comment]
Here's a gist that wraps up the css into a bookmarklet. both bookmarklet code, and steps to reproduce are there (don't trust random developers on the internet)

https://gist.github.com/gregdaynes/69a4fd362b65258f943a50b8dfc760f1
